### PR TITLE
Release v47.0.0

### PR DESCRIPTION
**Pull request by gingerbenw**
## v47.0.0 (2022-11-18)

This release adds support for expo 47

### CODE REVIEW

File: packages/expo/package.json [hunk 1]
```diff
@@ -49,11 +49,11 @@
     "bugsnag-build-reporter": "^1.0.1"
   },
   "devDependencies": {
-    "expo-constants": "~13.2.3"
+    "expo-constants": "~14.0.2"
   },
   "peerDependencies": {
-    "expo": ">=33.0.0",
-    "expo-constants": "~13.2.3",
+    "expo": "~47.0.3",
```
Comment (gingerbenw) [inline packages/expo/package.json]: only thing I'm unsure about here is the expo peer dependency version

Comment (imjoehaines) [reply packages/expo/package.json]: I think either leave it as it was (not sure why it was left at `>=33.0.0` previously — maybe just to reduce the number of changes necessary for a new version?) or change it to `^47.0.0`

**Review by tomlongridge — APPROVED** [review]

File: packages/expo/package.json (hunk 1 above)
Comment (imjoehaines) [inline packages/expo/package.json]: I don't think it makes sense to be this restrictive — why would we not allow `47.0.0` or a hypothetical `47.1.0`?

**Review by imjoehaines — CHANGES_REQUESTED** [review]

File: packages/expo/package.json (hunk 1 above)
Comment (kennethlynne) [reply packages/expo/package.json]: I'd like to suggest using `47.x.x` notation also to make it easier to understand what to expect

Comment (gingerbenw) [reply packages/expo/package.json]: have updated to `^47.0.0` to include all minor and patch versions - and have stuck with this semantic versioning syntax to keep consistency across all of our dependencies

**Review by imjoehaines — APPROVED** [review]

**Pull request merged**